Project.toml

```diff
@@ -1,18 +1,20 @@
 name = "WriteVTK"
 uuid = "64499a7a-5c06-52f2-abe2-ccb03c286192"
 authors = ["Juan Ignacio Polanco <jipolanc@gmail.com>"]
-version = "1.17.1"
+version = "1.18.0"
 
 [deps]
 Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
 CodecZlib = "944b1d66-785c-5afd-91f1-9de20f533193"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
 LightXML = "9c8b4983-aa76-5018-a973-4c85ecc9e179"
 TranscodingStreams = "3bb67fe8-82b1-5028-8e26-92a6c54297fa"
+VTKBase = "4004b06d-e244-455f-a6ce-a5f9919cc534"
 
 [compat]
 CodecZlib = "0.7"
 FillArrays = "0.13"
 LightXML = "0.9"
 TranscodingStreams = "0.9"
+VTKBase = "0.1"
```

Comment:
Can I suggest releasing 1.0.0 instead? It is more flexible to work with 3 numbers instead of just 2.

Reply:
Yes, good idea.